tests/sentry/spans/consumers/process/test_consumer.py

```diff
@@ -93,7 +96,80 @@ def add_commit(offsets, force=False):
             fac._flusher.join()
 
 
-@override_options({**DEFAULT_OPTIONS, "spans.drop-in-buffer": []})
+@override_options(
+    {**DEFAULT_OPTIONS, "spans.drop-in-buffer": [], "spans.process-segments.schema-validation": 0.0}
+)
+@pytest.mark.parametrize(
+    "field_to_set_none",
+    ["end_timestamp", "start_timestamp", "trace_id", "span_id"],
```

Comment:
We could also add a test or parameter for `end_timestamp = "some string"` (see INC-1458).